app/packs/src/apps/mydb/elements/details/reactions/variationsTab/ReactionVariationsAnalyses.js

```diff
@@ -0,0 +1,184 @@
+  }
+  return (
+    <div
+      className="custom-tooltip"
+      style={{
+        padding: '3px 8px',
+        color: '#fff',
+        backgroundColor: '#000',
+        borderRadius: '4px',
+      }}
+    >
+      Linked analyses:
+      <ul style={{ paddingLeft: '15px' }}>
+        {analyses.map((analysis) => (
+          <li key={analysis.id}>{analysis.name}</li>
+        ))}
+      </ul>
+
+    </div>
+  );
+}
+
+AnalysisOverlay.propTypes = {
+  value: PropTypes.instanceOf(AgGridReact.value).isRequired,
+};
+
+function AnalysisVariationLink({ reaction, analysisID }) {
+  const { variations } = cloneDeep(reaction);
+  const linkedVariations = variations.filter((row) => row.analyses.includes(analysisID)) ?? [];
+
+  if (linkedVariations.length > 0) {
+    return (
+      <Label
+        bsStyle="info"
+        onClick={() => UIActions.selectTab({ type: 'reaction', tabKey: 'variations' })}
+      >
+        {`Linked to ${linkedVariations.length} variation(s)`}
+        {' '}
+        <i className="fa fa-external-link" />
+      </Label>
+    );
+  }
+  return null;
+}
+
+AnalysisVariationLink.propTypes = {
+  reaction: PropTypes.instanceOf(Reaction).isRequired,
+  analysisID: PropTypes.string.isRequired,
+};
+
+function AnalysesCellRenderer({ value: analysesIDs }) {
+  return (
+    <div>{`${analysesIDs.length} link(s)`}</div>
+  );
+}
+
+AnalysesCellRenderer.propTypes = {
+  value: PropTypes.instanceOf(AgGridReact.value).isRequired,
+};
+
+function AnalysesCellEditor({
+  data: variationsRow,
+  value: analysesIDs,
+  onValueChange,
+  stopEditing,
+  context
+}) {
+  const [selectedAnalysisIDs, setSelectedAnalysisIDs] = useState(analysesIDs);
+  const { reactionShortLabel, allReactionAnalyses } = context;
+
+  const onAnalysisSelectionReady = () => {
+    onValueChange(selectedAnalysisIDs);
+    stopEditing();
+  };
+
+  const onChange = (analysisID) => {
+    if (selectedAnalysisIDs.includes(analysisID)) {
+      setSelectedAnalysisIDs(selectedAnalysisIDs.filter((id) => id !== analysisID));
+    } else {
+      setSelectedAnalysisIDs([...selectedAnalysisIDs, analysisID]);
+    }
+  };
+
+  const navigateToAnalysis = (analysisID) => {
+    UIActions.selectActiveAnalysis({
+      type: 'reaction',
+      analysisIndex: allReactionAnalyses.findIndex((analysis) => analysis.id === analysisID)
+    });
+    UIActions.selectActiveAnalysisTab(4.1);
+    UIActions.selectTab({ type: 'reaction', tabKey: 'analyses' });
+    stopEditing();
+  };
+
+  const analysesSelection = (
+    <div style={{ maxHeight: '200px', overflowY: 'auto' }}>
+      <FormGroup>
+        {allReactionAnalyses.filter((analysis) => !analysis.is_deleted).map((analysis) => (
+          <div key={analysis.id} style={{ display: 'flex', alignItems: 'center' }}>
+            <Checkbox
+              onChange={() => onChange(analysis.id)}
+              checked={selectedAnalysisIDs.includes(analysis.id)}
+              style={{ marginRight: '10px' }}
```

Comment:
Shall style be moved to CSS?

Reply:
I prefer to keep small, localized style stuff inline. I tell myself it's easier to maintain, but that might just be an excuse for not bothering to add a `.css` file. :see_no_evil:

Reply:
I am neither a front-end expert nor have been part of the Chemotion team for long enough to know the right approach here - so this was rather a question. It just looked odd to me to see styling here. But I can also understand that sometimes it's easier to keep things close together. So consider this resolved!